test/integration/protocol_integration_test.cc

```diff
@@ -501,6 +500,11 @@ TEST_P(ProtocolIntegrationTest, Retry) {
         TestUtility::findCounter(stats, "cluster.cluster_0.http2.tx_reset");
     ASSERT_NE(nullptr, counter);
     EXPECT_EQ(1L, counter->value());
+  } else if (upstreamProtocol() == FakeHttpConnection::Type::HTTP3) {
+    Stats::CounterSharedPtr counter =
+        TestUtility::findCounter(stats, "cluster.cluster_0.upstream_rq_tx_reset");
```

Comment:
do we not want a http3 stats scope to match the other branches here?

Reply:
I think landing this will help a few tests over in #15424 so would prefer to fix in a follow-up.  added TODO!